go/vt/vttablet/tabletmanager/action_agent.go

```diff
@@ -652,6 +657,7 @@ func (agent *ActionAgent) checkTabletMysqlPort(ctx context.Context, tablet *topo
 	}
 
 	if mport == topoproto.MysqlPort(tablet) {
+		agent.gotMysqlPort = true
```

Comment:
Hum this is a tricky one: there are two locks in the action agent: the action lock, and agent lock. We use the Locked suffix for the agent lock (to update the _XXX fields). We never use the Locked suffix for actionMutex. If we were, half the methods in the directory would have to change.

I'll just add a comment in the method, but won't change the method name. OK?